javascript/javascript1/week2/full-name.js

```diff
@@ -0,0 +1,8 @@
+function getFullname(firstname,surname){
+    return firstname+" "+surname
+  }
```

Comment:
There is some indenting here that is "off" (along with all the following lines).